src/main/java/edu/kit/hci/soli/domain/Room.java

```diff
@@ -0,0 +1,22 @@
+package edu.kit.hci.soli.domain;
+
+import edu.kit.hci.soli.controller.BookingsController;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToOne;
+import lombok.Data;
+
+/**
+ * The datamodel for a room as it is stored in the database
+ */
+@Entity
+@Data
+public class Room {
+
+    @Id
+    private Long id;
+
+    @OneToOne
+    private Booking currentBooking;
```

Comment:
Can a room have only one booking?

Reply:
Yes a Room can only store one currentBooking

Reply:
I think the table for bookings should contain a foreign key to the room table, otherwise having a room table is kind of redundant.